### ISSUE TITLE
Added methods for custom target namespace

### ISSUE THREAD

**Issue by weierophinney** [laminas/laminas-soap#13]
I need set custom target namespace
- https://github.com/zendframework/zend-soap/issues/24

---

Originally posted by @Sufir at https://github.com/zendframework/zend-soap/pull/34

**Comment by weierophinney** [comment]
@Sufir As noted by @adamlundrigan in October, 2016, we need documentation demonstrating how to use this new feature before we can merge it.

Additionally, we need you to rebase against current master to ensure we merge correctly.

Thanks!

---

Originally posted by @weierophinney at https://github.com/zendframework/zend-soap/pull/34#issuecomment-361318163

**Comment by weierophinney** [comment]
This would be useful for a project I'm currently working on!

@Sufir @weierophinney any objections if do a new PR with this + tests and docs?

---

Originally posted by @kynx at https://github.com/zendframework/zend-soap/pull/34#issuecomment-379234640

**Comment by weierophinney** [comment]
@kynx Go for it.

---

Originally posted by @weierophinney at https://github.com/zendframework/zend-soap/pull/34#issuecomment-379763828

**Comment by weierophinney** [comment]
> This package is considered feature-complete, and is now in **security-only** maintenance mode, following a [decision by the Technical Steering Committee](https://github.com/laminas/technical-steering-committee/blob/2b55453e172a1b8c9c4c212be7cf7e7a58b9352c/meetings/minutes/2020-08-03-TSC-Minutes.md#vote-on-components-to-mark-as-security-only).
> If you have a security issue, please [follow our security reporting guidelines](https://getlaminas.org/security/).
> If you wish to take on the role of maintainer, please [nominate yourself](https://github.com/laminas/technical-steering-committee/issues/new?assignees=&labels=Nomination&template=Maintainer_Nomination.md&title=%5BNOMINATION%5D%5BMAINTAINER%5D%3A+%7Bname+of+person+being+nominated%7D)
>
> If you need an actively developed SOAP client, we recommend [phpro/soap-client](https://github.com/phpro/soap-client).
